### ISSUE TITLE
REST API shows different structure for one endpoint

### ISSUE THREAD

**Issue by timwis** [awm33/taskflow#17]
`/tasks`, `/workflows` and `/workflow-instances` shows a structure like this:

```json
{
    "total_pages": 1,
    "page": 1,
    "count": 26,
    "data": [
        { ... }
    ]
}
```

But `/workflow-instances/recurring-latest` doesn't have that structure and just shows the data array:

```json
[
     { ... }
]
```

Not sure if this is intentional or not. One way I've seen around returning pagination details in the body (and having to wrap the actual body) is to do it in the response headers.

**Comment by awm33** [comment]
@timwis `/workflow-instances/recurring-latest` does not paginate

Something that may make it more consistent is to return:

```
{
    "count": 26,
    "data": [
        { ... }
    ]
}
```
Keeps a more simpler structure without making it seem paginated

**Comment by timwis** [comment]
Yep, or putting the count/page/total values in response headers and just keeping the body as the data.
